content/miq_dialogs/miq_provision_ibm_dialogs_template.yaml

```diff
@@ -16,7 +16,7 @@
       :description: Customization
       :fields:
         :ip_addr:
-          :description: Specify an IPv4 address
+          :description: Specify an IPv4 address  #TODO: Apply Gettext?
```

Comment:
Is there a way to apply gettext to these yaml templates? I couldn't find any examples from templates defined in other MIQ providers.

Reply:
This happens automatically in core.

Reply:
Coolness, thanks @Fryguy. I removed the `#TODO`.